.github/stale.yml

```diff
@@ -0,0 +1,12 @@
+daysUntilStale: 60
+daysUntilClose: 7
+exemptLabels:
+  - pinned
+  - security
+staleLabel: wontfix
```

Comment:
why `wontfix`?

Reply:
Seems like a fitting label. Open to suggestions though.

Reply:
how about "closed-stale" or something similar?